### PR TITLE
[RELEASE-4.10] Bug 2095667: Etcd readiness probe leaves many zombie process after heavy load

### PR DESCRIPTION
**Pull request by EmilyM1**
This has the readiness updates that Mustafa had made to 4.10 pulled in and the tcp updates

### CODE REVIEW

File: bindata/etcd/pod.yaml [hunk 1]
```diff
@@ -193,15 +193,13 @@ ${COMPUTED_ENV_VARS}
...
-      httpGet:
-        port: 9980
-        path: readyz
-        scheme: HTTPS
-      initialDelaySeconds: 10
-      timeoutSeconds: 10
-      failureThreshold: 3
-      periodSeconds: 5
-      successThreshold: 1
+    tcpSocket:
```
Comment (tjungblu) [inline bindata/etcd/pod.yaml:196]: I posted this on slack already, but two general questions:
* why do we need a TCP probe when we have a working http probe? The zombies came from the forking of the etcdctl process, which doesn't exist anymore
* your indentation actually removes the probe configuration, I reckon that's also why the build is green :)

**Review by tjungblu — CHANGES_REQUESTED** [review]

File: bindata/etcd/pod.yaml (hunk 1 above)
Comment (EmilyM1) [reply bindata/etcd/pod.yaml:196]: I can try it with http. I had read a few places that tcp was best practices. Let me put back the working http, as I see what is going on with the indentation.

**Pull request closed without merging**